modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -1,66 +1,79 @@
-|Flows are aggregated using a Max Concurrent model. The usage for a month is the highest number of flows that exist in a single given hour during a month.
+To calculate usage, MuleSoft tracks GB usage and aggregates the total GBs on a daily and monthly basis.
+// end::intro[]
 
-|_Mule message_: The container of the core information processed by the runtime. 
-|A Mule message counts as a single unit when an event source triggers it. Messages are aggregated using a sum of all messages sent during a month.
-|_Data throughput_: The total amount of data transferred in and out of the infrastructure that runs Mule where the Mule app is deployed. 
-|Data throughput counts when the deployed application transfers data to execute its business logic, including but not limited to internal operational network traffic for monitoring, logs, and health checks. Data throughput is aggregated as a sum of all bytes during a month. 
+== Requirements for Mule Runtime to Report Usage
 
-|API Manager	
-|_API instance under management_: API instances that are managed by API Manager after they are created using add, promote, or import options. 
-|API instances remain under management until they are deleted. Instances of API Manager are aggregated using a Max Concurrent model. The usage for a month is the highest number of APIs Managed in a single given hour during a month.
+// tag::supportedRuntimes[]
 
-|API Governance	
-|_API under Governance_: APIs that are identified by the selection criteria of at least one of the governance profiles. 
-|If an API is governed, all versions of that API are considered one governed API. Instances of API Governance are aggregated using a Max Concurrent model. The usage for a month is the highest number of APIs governed in a single given hour during a month.
+include::partial$usage-runtime.adoc[]
 
-|Flex Gateway	
-|_Flex Gateway API call_: Any access request received by Anypoint Flex Gateway regardless of whether the response to the request is successful or not. 
-|Flex Gateway requests are aggregated as a sum of all requests during a month.
+You must redeploy your apps with a supported version, either manually or during the monthly patching cycle. The monthly patching cycle is applicable only for CloudHub and CloudHub 2.0 applications. 
 
-|Composer	
-|_Composer task_: Any action executed on a Composer connector, including but not limited to read, create, update, and delete. 
-|Composer tasks are aggregated as a sum of all actions during a month.
+To xref:cloudhub-2::ch2-update-apps.adoc#roll-back-to-a-previous-successful-configuration[roll back] to a previous successful configuration of existing applications that use a Mule version that usage reports don't support, you must choose a compliant version before deploying changes. After deployment, a new configuration with the previously chosen settings and a supported runtime is created.
```

Comment:
The first sentence is a doozy - maybe split into two? Should it be singular "existing app"?